docs/java-installation.md

```diff
@@ -160,11 +160,105 @@ After this, you can run your tests as you normally do, for example using the `sb
 
 The following environment variables need to be configured when instrumenting your tests or application:
 
-| Environment variable | Default value     | Description                                                       |
-| -------------------- | ----------------- | ----------------------------------------------------------------- |
-| `$SCOPE_DSN`         |                   | Data source name (DSN) of Scope to be used when reporting results |
-| `$SCOPE_COMMIT_SHA`  | Autodetected (\*) | Commit hash to use when sending data to Scope                     |
-| `$SCOPE_REPOSITORY`  | Autodetected (\*) | Repository URL to use when sending data to Scope                  |
-| `$SCOPE_SOURCE_ROOT` | Autodetected (\*) | Repository root path                                              |
+| Environment variable | Default value           | Description                                                       |
+| -------------------- | ----------------------- | ----------------------------------------------------------------- |
+| `$SCOPE_DSN`         |                         | Data source name (DSN) of Scope to be used when reporting results |
+| `$SCOPE_COMMIT_SHA`  | Autodetected (\*)       | Commit hash to use when sending data to Scope                     |
+| `$SCOPE_REPOSITORY`  | Autodetected (\*)       | Repository URL to use when sending data to Scope                  |
+| `$SCOPE_SOURCE_ROOT` | Autodetected (\*)(\*\*) | Absolute filepath of the repository workspace                     |
 
 (\*) Autodetection of git information works if either tests run on a [supported CI provider](java-compatibility.md#ci-providers).
+
+(\*\*) If `$SCOPE_SOURCE_ROOT` cannot be autodetected based on the CI environment variables, it will be set to the folder path where `.git` folder is kept. In case `.git` folder does not exist, it will be set to the current folder path.
+
+### Run tests in a container.
+
+If you are running your tests in a container, forward the following environment variables based on your CI provider.
+
+> Take into account that `$SCOPE_SOURCE_ROOT` might need to be set **manually** to use the correct path inside the container.
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!-- Jenkins -->
+
+- `$SCOPE_DSN`
+- `$GIT_URL`
+- `$GIT_COMMIT`
+- `$WORKSPACE`
```

Comment:
We should not inject this, as it will take precedence over the auto detection of the source root. Same for the equivalent env vars for other CIs